react/components/ProductImages/components/Carousel/index.js

```diff
@@ -183,12 +183,14 @@ class Carousel extends Component {
 
       case 'video':
         return (
-          <Video
-            url={slide.src}
-            setThumb={this.setVideoThumb(i)}
-            playing={i === this.state.activeIndex}
-            id={i}
-          />
+          i === this.state.activeIndex && (
```

Comment:
I would render an empty div or a similar object in case this isn't true. This prevents the number of items in the carousel changing unexpectedly.